main.go

```diff
@@ -108,3 +106,16 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func checkEnvVar(name string) string {
+	value, isSet := os.LookupEnv(name)
+	if !isSet {
+		setupLog.Error(nil, "env variable must be set", "name", name)
+		os.Exit(1)
+	}
+	if value == "" {
```

Comment:
No need to check this now, isSet is enough

Reply:
done